pkg/operator/logutil/log.go

```diff
@@ -13,13 +13,13 @@ import (
 
 // Wrap wraps a log.Logger into a logr.Logger.
 func Wrap(l log.Logger) logr.Logger {
-	return &goKitLogger{l: l}
+	return logr.New(&goKitLogger{l: l})
```

Comment:
logr.Logger changed from an interface to a struct so we had to rework this file to make everything compile again.